samples/serial/README.md

```diff
@@ -0,0 +1,150 @@
+
+Use hub4com from the [com0com project](https://sourceforge.net/projects/com0com/files/hub4com/2.1.0.0/) on the server-side. We setup hub4com to associate the network port with the physical serial port, hub4com listens to the assigned port, establishes a connection to the client, and relays data from network port to physical port.
+
+Here is an example EFLOW configuration
+
+- Host ip = **172.18.246.137**
+- Host tcp port = **5002**
+- Host physical serial port = **COM3**
+
+- Client virtual serial port name = **/dev/ttyVirtS0**
+
+
+### Setup Host
+- Download and extract hub4com-2.1.0.0-386.zip from [com0com project](https://sourceforge.net/projects/com0com/files/hub4com/2.1.0.0/) to a local directory, e.g. c:\hub4com
+- Pick a TCP port number to associate to the serial port planned to expose.  In our example environment, we use port number 5002, and the port is associated with COM3
+    > If you have a firewall enabled, explicitly configure the firewall rule to allow inbound traffic for TCP port 5002.  Or run the command in the next step and allow access when asked.
+- Run hub4com in server mode, open a command prompt, type `cd /d c:\hub4com` and `com2tcp.bat \\.\COM3 5002` where **COM3** is the physical serial port name and **5002** is the TCP port number.  Click 'Allow access' if the Windows Defender Firewall dialog box pops up and asks for port access.
+- Hub4Com starts in server mode waiting for a connection.
+	
+
+### Setup Client
+- Connect EFLOW VM via Windows Admin Center or Powershell.  Check [Create a new deployment](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge-on-windows?view=iotedge-2018-06&tabs=powershell#create-a-new-deployment) on how to create your deployment of EFLOW on your target device.
+- Run socat in client mode: from the WAC CLI or Powershell window, type `sudo socat pty,link=/dev/ttyVirtS0,raw,user=iotedge-user,group=dialout,mode=777 tcp:172.18.246.137:5002`.  The command create a virtual serial port (**/dev/ttyVirtS0**) and relay data between the virtual serial port and server (ip address **172.18.246.137**, port **5002**)
+
+### Verify connection
+After the setup complete, you can verify the connection by sending data to the virtual serial port on the client and see if the data is received on the server.  The command window running hub4com will show messages when data received from or sent to the physical serial port.
+
+### Start the connection when system boot up
+You can further configure your host to enable the serial over network server when system boot up if necessary.
```

Comment:
**If necessary** , you can further configure your host to enable the serial over network server when system **boots** up.

Reply:
fixed